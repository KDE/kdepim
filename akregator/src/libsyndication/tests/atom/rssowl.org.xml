<?xml version="1.0" encoding="utf-8"?>
<!-- Source: http://www.rssowl.org
     Properties: 
      - Atom 0.3 with HTML content in CDATA section
-->
<feed version="0.3" xmlns="http://purl.org/atom/ns#">
  <title>RSSOwl - A Java RSS / RDF / Atom Newsreader - May the owl be with you</title>
  <link rel="alternate" type="text/html" href="http://www.rssowl.org"/>
  <modified>2005-08-12T16:27:23+02:00</modified>
  <entry>
    <title>Article on RSSOwl in german Java Magazin</title>
    <link rel="alternate" type="text/html" href="http://www.rssowl.org/node/190" />
    <id>http://www.rssowl.org/node/190</id>
    <issued>2006-03-08T22:11:04+01:00</issued>
    <modified>2006-03-08T22:15:15+01:00</modified>
    <content type="text/html" mode="escaped"><![CDATA[I wrote an article about RSSOwl for the popular german <a href="http://www.javamagazin.de/">Java Magazin</a>. It was added to this months issue and the article is even featured on the front page.
<br><br>
More information is available <a href="http://www.javamagazin.de/itr/ausgaben/psecom,id,305,nodeid,20.html">here</a>.
<br><br>
Ben]]></content>
    <author>
      <name>bpasero</name>
    </author>
  </entry>
  <entry>
    <title>RSSOwl 1.2.1 Beta-Test *update*</title>
    <link rel="alternate" type="text/html" href="http://www.rssowl.org/node/189" />
    <id>http://www.rssowl.org/node/189</id>
    <issued>2006-02-19T13:37:19+01:00</issued>
    <modified>2006-02-26T15:12:51+01:00</modified>
    <content type="text/html" mode="escaped"><![CDATA[It's been a while since the last beta-test, but here we go again with fresh new version RSSOwl 1.2.1, which I am about to release in the next couple of weeks.<br>
It includes a lot of bugfixes and some really cool new features that improve usability.<br><br> 

The preview is online for all operating systems at:
<ul>
<li>
<a href="http://www.rssowl.org/dl/Integration_Build/">http://www.rssowl.org/dl/Integration_Build/ </a>
</li>
</ul>
If you are interested in testing each bit of the new features and bugfixes, please follow the testplan at:
<ul>
<li>
<a href="http://www.rssowl.org/dl/Integration_Build/testplan.txt ">http://www.rssowl.org/dl/Integration_Build/testplan.txt </a>
</li>
</ul>
If you are interested in just using RSSOwl 1.2.1 preview, checkout the changelog at: 
<ul>
<li>
<a href="http://www.rssowl.org/dl/Integration_Build/changelog.txt ">http://www.rssowl.org/dl/Integration_Build/changelog.txt </a>
</li>
</ul>
If you encounter any bug or issue, please directly contact me via <a href="http://www.rssowl.org/contact">eMail</a> or join our IRC channel #rssowl at irc.freenode.net.
<br><br>
<b>Update</b>: Uploaded a new Beta (<a href="http://www.rssowl.org/dl/Integration_Build/changelog.txt">Changelog</a>).<br>]]></content>
    <author>
      <name>bpasero</name>
    </author>
  </entry>
</feed>
