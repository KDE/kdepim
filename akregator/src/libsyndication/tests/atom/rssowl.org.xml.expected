# Feed begin ######################
title: #RSSOwl - A Java RSS / RDF / Atom Newsreader - May the owl be with you#
link: #http://www.rssowl.org#
# Item begin ######################
id: #http://www.rssowl.org/node/190#
title: #Article on RSSOwl in german Java Magazin#
link: #http://www.rssowl.org/node/190#
content: #I wrote an article about RSSOwl for the popular german <a href="http://www.javamagazin.de/">Java Magazin</a>. It was added to this months issue and the article is even featured on the front page. <br><br> More information is available <a href="http://www.javamagazin.de/itr/ausgaben/psecom,id,305,nodeid,20.html">here</a>. <br><br> Ben#
datePublished: #Wed Mar 8 22:11:04 2006#
dateUpdated: #Wed Mar 8 22:11:04 2006#
# Person begin ####################
name: #bpasero#
# Person end ######################
# Item end ########################
# Item begin ######################
id: #http://www.rssowl.org/node/189#
title: #RSSOwl 1.2.1 Beta-Test *update*#
link: #http://www.rssowl.org/node/189#
content: #It's been a while since the last beta-test, but here we go again with fresh new version RSSOwl 1.2.1, which I am about to release in the next couple of weeks.<br> It includes a lot of bugfixes and some really cool new features that improve usability.<br><br> The preview is online for all operating systems at: <ul> <li> <a href="http://www.rssowl.org/dl/Integration_Build/">http://www.rssowl.org/dl/Integration_Build/ </a> </li> </ul> If you are interested in testing each bit of the new features and bugfixes, please follow the testplan at: <ul> <li> <a href="http://www.rssowl.org/dl/Integration_Build/testplan.txt ">http://www.rssowl.org/dl/Integration_Build/testplan.txt </a> </li> </ul> If you are interested in just using RSSOwl 1.2.1 preview, checkout the changelog at: <ul> <li> <a href="http://www.rssowl.org/dl/Integration_Build/changelog.txt ">http://www.rssowl.org/dl/Integration_Build/changelog.txt </a> </li> </ul> If you encounter any bug or issue, please directly contact me via <a href="http://www.rssowl.org/contact">eMail</a> or join our IRC channel #rssowl at irc.freenode.net. <br><br> <b>Update</b>: Uploaded a new Beta (<a href="http://www.rssowl.org/dl/Integration_Build/changelog.txt">Changelog</a>).<br>#
datePublished: #Sun Feb 19 13:37:19 2006#
dateUpdated: #Sun Feb 19 13:37:19 2006#
# Person begin ####################
name: #bpasero#
# Person end ######################
# Item end ########################
# Feed end ########################

